components/organisms/Layout.js

```diff
@@ -23,7 +26,18 @@ export const Layout = ({
   return (
     <div className="overflow-x-hidden">
       <header>
-        <div className="layout-container ">
+        {phaseText ? (
+          <PhaseBanner phase={phaseTitle}>{phaseText}</PhaseBanner>
```

Comment:
Think this is pretty static across all pages, not sure if we need to pass it in for each page